packer/packer-virtualbox.json

```diff
@@ -45,13 +45,13 @@
           "modifyvm",
           "{{.Name}}",
           "--memory",
-          "384"
+          "4096"
         ],
         [
           "modifyvm",
           "{{.Name}}",
           "--cpus",
-          "1"
+          "4"
```

Comment:
I don't think we want to increase the resource limits here, because this is just for the Packer image build environment, which doesn't need much (just enough to do apt-get install, etc.)